src/Components/PresentationalComponents/CSAwRuleBox/CSAwRuleBox.js

```diff
@@ -105,7 +146,7 @@ const CSAwRuleBox = ({ rules, synopsis, changeExposedSystemsParameters, intl })
                                     <StackItem>
                                         <TextContent>
                                             <Text
-                                                className={'icon-with-label'}
+                                                className='icon-with-label pf-u-mt-xs'
```

Comment:
I think we need to define or set a rule about the single quote `''` or double-quote `""` which one do we want to use for the `className`